libraries/bsg_manycore_responder.h

```diff
@@ -150,10 +150,11 @@ int hb_mc_responder_del(hb_mc_responder_t *responder);
 					throw r;			\
 			}						\
 			~__bsg_manycore_ ## rspdr ## _add_del () {	\
-				int r = hb_mc_responder_del(&rspdr);	\
-				if (r != HB_MC_SUCCESS)			\
-					throw r;			\
-			}						\
+                                int r = hb_mc_responder_del(&rspdr);    \
+                                if (r != HB_MC_SUCCESS)                                       \
+                                        bsg_pr_err("%s: failed to remove responder %s: %s\n", \
+                                                    __FILE__, #rspdr, hb_mc_strerror(r));      \
+			}					\
```

Comment:
Can you file a separate PR for this?

Reply:
This is awesome, but not really part of the PR

Reply:
OK

Reply:
You might need to rebase on master

Reply:
OK, rebased!